### PR TITLE
Routing issue fixed.

### PR DESCRIPTION
**Pull request by abhishekshree**
This is a PR for #14 and it does the following:

- Created an App.js and separated Routing logic from index.js
- Used HashRouter instead of BrowserRouter

This is a well know problem in and this approach for fixing it is based on the docs, [here](https://create-react-app.dev/docs/deployment/#notes-on-client-side-routing)

Also, it is important to specify the ```"homepage": " "``` in ```package.json``` to our base website url before running the command ```npm run deploy```

More details on this problem can be found on StackOverflow, [webpack approach](https://stackoverflow.com/questions/43209666/react-router-v4-cannot-get-url) and [a very well explained reason doc.](https://stackoverflow.com/questions/27928372/react-router-urls-dont-work-when-refreshing-or-writing-manually)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+  "homepage": "",
```
Comment (shubhamkorde) [inline package.json:2]: If this field is empty, better remove it.

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+var path = require("path");
```
Comment (shubhamkorde) [inline webpack.config.js:1]: Do we still need webpack config to solve this issue after using HashRouter. If yes, why? If not, we should clean up this file and npm dependencies.

**Review by shubhamkorde — COMMENTED** [review]
Looks mostly good. Just a clarification.

File: webpack.config.js (hunk 2 above)
Comment (abhishekshree) [reply webpack.config.js]: Yes this was redundant after HashRouter, I've removed that and updated dependencies.

**Review by shubhamkorde — APPROVED** [review]

**Pull request merged**